src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -75,7 +75,7 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
   }
 
   @Override
-  public ResponseEntity<ApiGcsBucketReference> getBucketReference(UUID id, UUID referenceId) {
+  public ResponseEntity<ApiGcsBucketReference> getGcpBucketReference(UUID id, UUID referenceId) {
```

Comment:
Well, the endpoint is
`resources/referenced/gcp/buckets/{resourceid}`
because we are talking about the cloud, not the resource type. In this case, I removed the gcs altogether.

I wish we hadn't used it anywhere, since GCS adds no information over GCP and will only be meaningful to GCP-savvy folks.

We should probably revisit the naming of resource components in the OpenAPI so they consistently include the cloud name. I added PF-605 to track this.

Reply:
It's the hexagon name, though. You include BigQuery with dataset, but no service name for bucket. Which is a small point, but they could always introduce another service that used buckets.

Reply:
"hexagon"?

I made ticket PF-605 to rename all of our GCP-specific api components with a Gcp prefix. I think that will help clarify what is cloud-specific.

Reply:
All the gcp services have hexagon logos. Or at least they did until about a year ago.